src/memcached/windowed_cache_impl.go

```diff
@@ -0,0 +1,160 @@
+package memcached
+
+import (
+	"context"
+	"math/rand"
+	"strconv"
+	"sync"
+
+	"github.com/bradfitz/gomemcache/memcache"
+	"github.com/coocood/freecache"
+	pb "github.com/envoyproxy/go-control-plane/envoy/service/ratelimit/v3"
+	"github.com/envoyproxy/ratelimit/src/algorithm"
+	"github.com/envoyproxy/ratelimit/src/config"
+	"github.com/envoyproxy/ratelimit/src/limiter"
+	"github.com/envoyproxy/ratelimit/src/memcached/driver"
+	"github.com/envoyproxy/ratelimit/src/utils"
+	logger "github.com/sirupsen/logrus"
+)
+
+type windowedRateLimitCacheImpl struct {
+	client                     driver.Client
+	timeSource                 utils.TimeSource
+	jitterRand                 *rand.Rand
+	expirationJitterMaxSeconds int64
+	localCache                 *freecache.Cache
```

Comment:
we no longer need this field in both window impls right? (as it is part of `WindowImpl` now)